pmd-apex/src/test/resources/net/sourceforge/pmd/lang/apex/rule/bestpractices/xml/ApexUnitTestMethodShouldHaveIsTestAnnotation.xml

```diff
@@ -26,7 +26,7 @@ private class ATest {
     static void methodETest() {
         System.debug('I am a debug statement.');
     }
-    static void methodFTest() {
+    static testmethod void methodFTest() {
         System.assertEquals(1,2);
     }
     private void fetchData() {
```

Comment:
Could you also add a simple test case for #3282? e.g.

```xml
    <test-code>
        <description>false positive with @testsetup #3282</description>
        <expected-problems>0</expected-problems>
        <code><![CDATA[
public class TestFactory {
    @testsetup static void setup(){
        System.assertEquals(null, null, 'this should be null');
    }
    @isTest static void testCase1() { }
}
]]></code>
    </test-code>
```
Thanks!

Reply:
Done, thanks for picking that up. Theoretically could have been added to an existing test, but I think it's a good practice to explicitly have it separate.